src/btree/bt_slvg.c

```diff
@@ -512,26 +511,6 @@ err:	__wt_free(session, trk->trk_addr);
 }
 
 /*
- * __slvg_trk_split --
- *	Split a tracked chunk.
- */
-static int
-__slvg_trk_split(WT_SESSION_IMPL *session, WT_TRACK *orig, WT_TRACK **newp)
```

Comment:
I thought about that, but `++a_trk->shared->ref;` is part of the work we're doing to split the WT_TRACK object, and I'd rather do that inline instead of in some other place.